bin/run_snd.py

```diff
@@ -7,13 +7,9 @@
 from imp import reload
 from pathlib import Path
 import warnings
-
-# from bluesky.preprocessors import run_wrapper
-
-# from hxrsnd.plans.scans import linear_scan
 from hxrsnd.utils import setup_logging
 
-# Ignore python warnings
+# Ignore python warnings (Remove when ophyd stops warning about 'signal_names')
```

Comment:
I think this issue may be fixed in newer versions of `ophyd`?

Reply:
Still there https://github.com/NSLS-II/ophyd/blob/master/ophyd/device.py#L756-L759